lib/iris/analysis/__init__.py

```diff
@@ -2114,8 +2117,8 @@ def interp_order(length):
 
 SUM = WeightedAggregator(
     "sum",
-    iris._lazy_data.non_lazy(_lazy_sum),
-    lazy_func=_build_dask_mdtol_function(_lazy_sum),
+    _sum,
+    lazy_func=_build_dask_mdtol_function(_sum),
```

Comment:
As for `_count`, worth a note explaining that `_sum` must and can act as a lazy operator.